buildSrc/src/main/groovy/com/uber/okbuck/rule/android/AndroidTestRule.groovy

```diff
@@ -40,6 +40,7 @@ final class AndroidTestRule extends AndroidRule {
                 aptDeps,
                 providedDeps,
                 aidlRuleNames,
+                null,
```

Comment:
I was actually waiting for your Buck PR to merge so I can verify it was working in tests.